config/config.go

```diff
@@ -228,6 +228,14 @@ type ScrapeConfig struct {
 	Scheme string `yaml:"scheme,omitempty"`
 	// The HTTP basic authentication credentials for the targets.
 	BasicAuth *BasicAuth `yaml:"basic_auth,omitempty"`
+	// The bearer token for the targets.
+	BearerToken string `yaml:"bearer_token,omitempty"`
```

Comment:
That's why I was asking. I honestly don't know if there are any protocols forbidding it or not.

If there are not, I agree with you.